BedrockServer.cpp

```diff
@@ -2009,7 +2011,6 @@ bool BedrockServer::shouldBackup() {
 }
 
 SData BedrockServer::_generateCrashMessage(const unique_ptr<BedrockCommand>& command) {
-    SHMMM("Generating CRASH_COMMAND command for " << command->request.methodLine);
```

Comment:
Can we please not remove this? I added this not long ago to help debug weird scenarios 😅 

I know you added logs in the callers of this method, but I feel like we might as well leave this one